masterdata/configuration/liquibase/liquibase.xml

```diff
@@ -499,4 +499,18 @@
         </sql>
     </changeSet>
 
+    <changeSet id="GOK-236-update-inv-StockRooms"  author="Inventory">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">select count(*) from location where name = 'Hosakote Namma Clinic'</sqlCheck>
```

Comment:
<sqlCheck expectedResult="1">select count(*) from location where name = 'Hosakote Namma Clinic'</sqlCheck>

Expected condition is not correct,  it should be 'Hosakote Nama Clinic'